Features/CaseSearch/test_pages/casesearch_page.py

```diff
@@ -10,14 +16,157 @@ class CaseSearchWorkflows(BasePage):
         self.case_name_format = "//tr[.//td[text()='{}']]"
+        self.text_search_property_name_and_value_format = "//input[contains (@id, '{}') and @value='{}']"
+        self.search_property_name_combobox = "//label[contains(text(),'{}')]"
+        self.combobox_search_property_name_and_value_format = self.search_property_name_combobox + "//following::span[contains(text(),'{}')]"
+        self.search_against_text_property_format = "//input[contains (@id, '{}')]"
+        self.help_text_format = '//label[@for="{}"]//following::a[@data-content="{}"]'
+        self.combox_select = "//label[contains(text(), '{}')]//following::select[contains(@class, 'query-field')][1]"
+        self.search_for_address = "//*[contains(text(),'{}')]//following::input[contains(@aria-label,'Search')][1]"
+        self.include_blanks = self.search_property_name_combobox + "//following::input[contains(@class,'search-for-blank')][1]"
+        self.required_validation_on_top = "//div[@class='alert alert-danger']//following::li[contains(text(),'{}')]"
+        self.required_validation_per_property_text = self.search_against_text_property_format + "//following::div[contains (text(),'{}')][1]"
+        self.required_validation_per_property_combox = self.search_property_name_combobox + "//following::div[contains (text(),'{}')][1]"
+        self.city_value_home = "//span[@class='caption webapp-markdown-output'][contains(text(), '{}')]"
+        self.city_value_work = "//span[@class='caption webapp-markdown-output'][contains(text()[2], '{}')]"
+        self.search_screen_title = "//h2[contains(text(), '{}')]"
+        self.date_selected = "(//span[@class='drp-selected' and contains(text(),'{}')])[1]"
+        self.dropdown_values = self.combox_select + "/option"
+        self.menu_header = "//h1[contains(text(),'{}')]"
+        self.menu_breadcrumb = "//li[contains(text(),'{}')]"
+        self.webapps_home = (By.XPATH, "//i[@class='fcc fcc-flower']")
+        self.case_detail_value = "//th[contains(text(), '{}')]//following-sibling::td[contains(text(), '{}')]"
+
+        # Reports
+        self.case_type_select = (By.ID, "report_filter_case_type")
+        self.report_search = (By.ID, "report_filter_search_query")
+        self.report_apply_filters = (By.ID, "apply-filters")
+        self.commcare_case_claim_case = "//a[contains(text(), '{}')]//following::*[text()='{}'][1]"
 
     def check_element_claimed(self, case_name):
         self.case = self.get_element(self.case_name_format, case_name)
         assert self.is_visible_and_displayed(self.case)
 
-    def check_default_rating_on_caselist(self, default_rating):
-        ratings = self.find_elements_texts(self.rating_in_table)
-        assert default_rating in ratings
+    def check_values_on_caselist(self, row_num, value, is_multi="NO"):
+        self.value_in_table = self.get_element(self.value_in_table_format, row_num)
+        values_ = self.find_elements_texts(self.value_in_table)
+        print(value, values_)
```

Comment:
The print statements are for debugging purposes. I've commented this one (since this will be required more often)
https://github.com/dimagi/dimagi-qa/pull/156/commits/c6134eb4a78a3ff9daf0f5ef4a8a4c575ac6bed3